src/Pages/NewsDetail.css

```diff
@@ -1,3 +1,6 @@
-.news-detail-img {
+.news-detail__image {
+    box-shadow: var(--card-shadow);
+    margin: -75px 0 2rem 0;
```

Comment:
use rem on `75px` ?

Reply:
The thing is that the header section is set in pixels (which is ok, we don't want that to grow with the font size) so this `-75px` is to put the news detail image on top of that section, so in order to make sure that this work no matter what I used px.